app/assets/javascripts/application.js

```diff
@@ -51,8 +51,8 @@
 //= require controllers/container_dashboard/card/card-module
 //= require controllers/container_dashboard/card/heatmaps/heatmaps-card-directive
 //= require controllers/container_topology/container_topology_controller
-//= require d3/d3
-//= require c3/c3
+//= require c3
+//= require d3
```

Comment:
ok, according to @himdel, this is actually better than the original but should not change anything